docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,82 @@
+
+Also called platform or internal authenticators, a secure enclave is physically bound to a specific computing device.
+
+- TPM (Trusted Platform Module): These devices are usually built into a product's mainboard, or can be installed in devices with a TPM header, as shown [here][toms-hardware-tpm]. They include a small processor to carry out cryptographic functions on the device, instead of on the system's processor where it could be interfered with. Trust is usually derived from a private key or certificate signed by a trusted manufacture's certificate authority.
+
+- Mobile devices: Most newer Apple and Android devices include a [Secure Enclave][apple-enclave] or [Hardware-backed Keystore][android-keystore]
+
+#### Hardware Security Keys
+
+Also known as cross-platform or roaming authenticators, these are authentication devices which can move with the user across different computers.
+
+- FIDO U2F: This [open standard][fido-spec] is used by many products like Yubico's [Yubikey][yubikey-products] and Google's [Titan Security Key](https://support.google.com/titansecuritykey/answer/9115487?hl=en). They usually secure a private key used to decrypt information signed by an accessible public key.
+
+::: tip Note
+The nature of cross-platform keys mean they are not associated with a single end-user device. Pomerium policies can be written to allow these keys, or specified to only accept secure enclaves.
+:::
+
+## Implement Device Identity with Pomerium
+
+Pomerium supports policies that use device identity since version [0.16.0](/docs/upgrading.md#policy-for-device-identity). We use the [Web Authentication][webauthn-api] (**WebAuthN**) API to bring authentication and authorization based on device identity into your security framework. It enables users to register their device ID, and admins to require a trusted device before accessing one, several, or all services.
+
+You can review our implementation of the WebAuthn specification [on GitHub](https://github.com/pomerium/webauthn).
+
+To get started, review the following pages:
+
+- [Pomerium Policy Language](/docs/topcics/ppl.md) to learn how to build policies that use device ID.
+- [Enroll a Device](/guides/enroll-device.md) to teach end-users how to enroll devices on Pomerium.
+
+
+## Looking Ahead: Device Posture
+
+Even if they only use the right device, what happens when a exploit is discovered in their OS or browser? How do you restrict access until the exploit is patched? Expanded iterations of device identity methods account for this using **device posture**, sometimes referred to as device state.
+
+A device state is more complex superset of device identity, with more information about the device and software being used to generate the resulting identifier. This includes the operating system and web browser version. Device posture can be used by secure systems to ensure that users do not access sensitive information using software that has not been updated for, say, a known security flaw.
```

Comment:
I think the previous paragraph now addresses the key points. Thoughts? 

```suggestion
```

Reply:
Mostly yes, but there's one part of this paragraph I want to include above before signing off on removing it. Will make a suggestion above.